source/components/cardContainer/cardContainer.html

```diff
@@ -26,15 +26,16 @@
 				<div *ngIf="hasItems$ | async">
 					<div class="card-item-repeat" *ngFor="let card of dataSource.dataSet$ | async">
 						<rlCard [item]="card"
-								[save]="save"></rlCard>
+								[save]="save"
+								[permanentFooter]="permanentFooters"></rlCard>
 					</div>
 				</div>
 
 				<div>
 					<rlBusy [loading]="dataSource.loadingDataSet$ | async" size="2x"></rlBusy>
 				</div>
 
-				<div class="card-container-footer" *ngIf="!(dataSource.isEmpty$ | async)">
+				<div class="card-container-footer">
```

Comment:
I added this back in and it didn't affect the changes I had made. Great catch Sam!